utils/astxml2markdown.xslt

```diff
@@ -13,6 +13,8 @@ xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 </xsl:template>
 
 <xsl:template match="application">
+    <xsl:text>Title: </xsl:text><xsl:value-of select="@name"/><xsl:text></xsl:text>
+    <xsl:text>&#10;&#10;</xsl:text>
```

Comment:
Can you add the `---` header separators so this looks like the static documentation?  I.E.
```
---
title: MyApp
---
```

Same for the functions